src/libsync/vfs/cfapi/cfapiwrapper.cpp

```diff
@@ -407,7 +411,7 @@ QString retrieveWindowsSid()
     return {};
 }
 
-bool createSyncRootRegistryKeys(const QString &providerName, const QString &folderAlias, const QString &displayName, const QString &accountDisplayName, const QString &syncRootPath)
+bool createSyncRootRegistryKeys(const QString &providerName, const QString &folderAlias, const QString &navigationPaneClsid, const QString &displayName, const QString &accountDisplayName, const QString &syncRootPath)
```

Comment:
nitpick: This is *a lot* of parameters, IMO a dedicated struct might be a good idea

Reply:
@claucambra I will do this in a separate PR. The struct must be shared between the `vfs.h` and `cfapiwrapper.cpp`, otherwise, if the structure is internal to `cfapiwrapper`, there still will be a lot of parameters in the function that calls `createSyncRootRegistryKeys`